app/index.js

```diff
@@ -44,7 +44,10 @@ module.exports = generators.Base.extend({
         type: 'input',
         name: 'app_id',
         message: 'App ID',
-        default: answers => answers.name
+        validate: (input) => {
+          return (/\-|\s/.test(input)) ? 'Invalid App ID, use alphanumeric and periods only' : true;
```

Comment:
Yes, I kept the validation 'light' because as stated in the commit message, I do not have an exact spec for what constitutes a valid App ID.

But perhaps a different approach here should be considered;

The issue I encountered was that accepting the default App ID resulted in a failure to build in the tabris-js.com online build tool. And at that time I had no idea why. It would seem that a similar experience is what lead to the creation of #8 

So the real goal is to match the validation rules used in this generator with those that are being applied to the tabris-js.com build tool.

@ralfstx, Are you able to get your hands on the exact regex being used to validate the `widget` tag's `id` attribute in `cordova/config.xml`?

I believe the rules are actually rather complex. for example, when creating App ID's in my Apple developer account, I notice that wildcards are permissible, EG: `com.tabrisjs.*`. 

But in any case, the ultimate measure of correctness would be a scenario where the derived (or validated user input) for App ID, is always be accepted by the online build tool.

In other words, the generator should use the build tool as the source of truth, after all, that is why we are validating the App ID in the first place.